presto-function-server/pom.xml

```diff
@@ -129,6 +149,30 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Instead of using the Pinot plugin and bringing in those dependencies, can you just create a simple local function and register it?